### ISSUE TITLE
Mouse & Keyboard USB not working login screen Windows 7 4.06

### ISSUE THREAD

**Issue by Javisky** [veyon/veyon#316]
Sorry for my english I'm from Spain

- After update to 4.06 suddenly after a reboot i cannot use keyboard or mouse on logon screen on windows 7. I cannot login to my account , **TOTAL FREEZE**

- It happened in 20 computers after a remote reboot (students) ,  and in mine (master one ).
- The keyboard leds not working, (Num Bloq...)
- Mouse red light it's on.

** I 've tried:**

- Plug in others USB (not working) 
- Use others mouse or keyboards USB (not working)
- Boot Safe mode ( not working)
- Boot last known good configuration _(working some computers)_
- Stop service (remotely) not working _(using remote desktop)_
- Change drivers    >  Neither windows update o stored Drivers _(using remote desktop)_

**Solution:**
- Thanks i have rpd configurated, I could use Remote desktop connection , install 4.05 and suddenly the mouse get to working, and the keyboard.. after remove and plug again.

**PS: the problem with devices, it's suddenly change status to 39 error, after reboot.**

En español:

- De repente tras reiniciar varios equipos e incluso el mío que es el único que tiene Master instalado, han empezado a bloquearse y no dejaban iniciar sesión de ninguna forma, ya que el teclado y el ratón han dejado de funcionar.

- He probado a reconectarlos .. a conectar otros y nada.. al final el problema era la versión 4.06 **tras volver a la 4.05 se ha arreglado el problema del ratón**. Y luego tras reconectar el teclado también este.

- He podido solucionarlo usando la conexión remota de windows desde otro PC, por suerte tenia acceso. 
- Otra opción de arreglarlo es si se tiene acceso con psexec a los equipos, se puede lanzar el comando para que se instale una versión anterior de veyon.
- En algunos Pc me ha dejado repararlo arrancando windows con la ultima versión buena conocida.. porque en modo a prueba de fallos no me funcionaba tampoco.
- Detener el servicio de veyon tampoco funcionaba.

Ver capturas

Un saludo

- 
![image 001](https://user-images.githubusercontent.com/38493355/38925386-c2f2ce86-42ff-11e8-9263-3f90e98dab54.png)
![image 002](https://user-images.githubusercontent.com/38493355/38925388-c30fb406-42ff-11e8-99a4-a3f26457f8fd.png)

**Comment by tobydox** [comment]
Thanks for reporting this important issue! There seems to be a serious issue with the update input device interception driver in 4.0.6. We'll switch back to the previous version which will be part of the 4.0.7 release which will be available later today.

**Comment by tobydox** [comment]
Please update to 4.0.7 to fix the problem.

**Comment by Na5050** [comment]
I have the same problem with windows 7 and veyon 4.1.1&4.1.0

**Comment by robotoja** [comment]
I also have the same problem on widnows 7 64bit and veyon 4.1.1.0

**Comment by bacsom** [comment]
I had the same issue on Win 7 64bit and Veyon 4.1.2., it seems that an uninstall and reinstall did the trick.

**Comment by robotoja** [comment]
Thank you for your answer. 
The problem appears within the Master and you don't have RDP enabled on it.

**Comment by Mjbert** [comment]
I jut downloaded 4.1.2 Windows 64 version and my usb mouse is not working. I tried the uninstall and reinstall of Veyon Master but it does not seem to be working. Does anyone know a solution to this.

**Comment by tobydox** [comment]
There's a known issue within the input device Interception driver. You can try to manually uninstall and install the Interception driver by opening a command line window with administrative privileges:

```
cd "C:\Program Files\Veyon"
install-interception /uninstall
```

and after reboot

```
cd "C:\Program Files\Veyon"
install-interception /install
```

**Comment by tobydox** [comment]
Issue is fixed in 4.1.5.
